### ISSUE TITLE
Support JSON:API spec

### ISSUE THREAD

**Issue by mhluska** [vuex-orm/vuex-orm#334]
It would be interesting if this supported the [JSON:API spec](https://jsonapi.org/). 

It seems like currently the format supported by vuex-orm is the default JSON serialization format of Rails? Currently I am converting my JSON:API responses to an intermediate format that vuex-orm understands which seems to negate the performance benefits of JSON:API.

**Comment by kiaking** [comment]
@mhluska Hi! Thank you for the feedback! I think if we have #326, we could do this. I'm not 100% sure if the core library should have JSON:API mapping feature built in (it might should be a plugin) but I agree that we should have some kind of way to map data structure.

**Comment by Uysim** [comment]
I hope we can have this feature

**Comment by kiaking** [comment]
Yeah, sorry it's taking a long time. I must get my feet back on the ground really...!

**Comment by carsomyr** [comment]
@kiaking @mhluska @Uysim Shameless ad: Can this issue be closed with the release of my vuex-orm-json-api package? https://www.npmjs.com/package/vuex-orm-json-api
